bin/que

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+require 'ostruct'
+require 'logger'
+
+options = OpenStruct.new
+options.worker_count = 4
+options.log_level = 'info'
+options.wake_interval = 0.1
+
+OptionParser.new do |opts|
+  opts.banner = 'usage: que [options] file/to/require ...'
+
+  opts.on('-w', '--worker-count [COUNT]', Integer, "Set number of workers in process (default: 4)") do |worker_count|
+    options.worker_count = worker_count
+  end
+
+  opts.on('-i', '--wake-interval [INTERVAL]', Float, "Set maximum interval between polls of the job queue (in seconds) (default: 0.1)") do |wake_interval|
+    options.wake_interval = wake_interval
+  end
+
+  opts.on('-l', '--log-level [LEVEL]', String, "Set level of Que's logger (debug, info, warn, error, fatal) (default: info)") do |log_level|
+    options.log_level = log_level
+  end
+
+  opts.on('-q', '--queue-name [NAME]', String, "Set the name of the queue to work jobs from (default: the default queue)") do |queue_name|
+    options.queue_name = queue_name
+  end
+
+  opts.on('-v', '--version', "Show Que version") do
+    $stdout.puts "Que version #{Version}"
+    exit 0
+  end
+
+  opts.on('-h', '--help', "Show help text") do
+    $stdout.puts opts
+    exit 0
+  end
+end.parse!(ARGV)
+
+if ARGV.length.zero?
+  $stdout.puts <<-OUTPUT
+You didn't include any Ruby files to require!
+Que needs to be able to load your application before it can process jobs.
+(Hint: If you're using Rails, try `que config/environment.rb`)
+(Or use `que -h` for a list of options)
+OUTPUT
+exit 1
+end
+
+ARGV.each do |file|
+  begin
+    # Need to add '.' to load path for relative requires
+    $LOAD_PATH << '.'
+    require file
```

Comment:
Generally this is a bad idea. People should just pass `./file.rb` if they want to do so. Ruby removed `.` from $LOAD_PATH for security reason, it's not a good idea to restore the old behaviour.